packages/tradershub/src/features/cfd/modals/MT5PasswordModal/SubmitHandler.jsx

```diff
@@ -0,0 +1,58 @@
+import { Category, MarketType, PlatformDetails } from '../../constants';
+import { Jurisdiction } from '../../screens/CFDCompareAccounts/constants';
+
+export const submitHandler = async (
+    isMT5PasswordNotSet,
+    tradingPasswordChange,
+    password,
+    marketType,
+    activeTrading,
+    mutate,
```

Comment:
tradingPasswordChange,
mutate,
activeTrading,
settings,

These are coming from `deriv/api` so we can directly call them here instead of passing props